### PR TITLE
[Debugger] Update debugger guide

### PR DESCRIPTION
**Pull request by praveennadarajah**
## Purpose
This PR will update the debugger guide with modified gifs.

Fixes https://github.com/ballerina-platform/ballerina-dev-website/issues/3171

### CODE REVIEW

**Comment by NipunaRanasinghe** [comment]
@praveennadarajah Shall we add a section on debugging individual tests using the newly added codelens support?

File: learn/guides/visual-studio-code-extension/debugging.md [hunk 1]
```diff
@@ -33,7 +33,21 @@ redirect_from:
 
 ## Starting a Debug Session
 
-The VS Code Ballerina extension gives you the same debugging experience as the conventional VS Code Debugger. Follow the steps below to start a debug session.
+The VS Code Ballerina extension gives you the same debugging experience as the conventional VS Code Debugger.
+
+Follow the steps below to start a quick debug session.
```
Comment (NipunaRanasinghe) [inline learn/guides/visual-studio-code-extension/debugging.md:38]:
```suggestion

### Using Codelense

Ballerina extension provides multiple options to debug your Ballerina program and the easiest way will be using our context-aware `debug` codelense support provided by the extension. The extension can detect any program entry points (main function/service definition) on the fly, and the codelense will appear automatically. Follow the steps below to start a debug session for a program with a main function, as shown below.
```

File: learn/guides/visual-studio-code-extension/debugging.md [hunk 2]
```diff
@@ -33,7 +33,21 @@ redirect_from:
...
+
+![Start Quick Debug Session](/learn/images/start-quick-debug-session.gif)
+
+<br/>
+
+1. Open the folder, which includes the Ballerina program you want to debug, and select the file.
```
Comment (NipunaRanasinghe) [inline learn/guides/visual-studio-code-extension/debugging.md:44]:
```suggestion
1. Open the folder, which includes the Ballerina program you want to debug, and open the source file in the editor.
```

File: learn/guides/visual-studio-code-extension/debugging.md [hunk 3]
```diff
@@ -33,7 +33,21 @@ redirect_from:
 
 ## Starting a Debug Session
 
-The VS Code Ballerina extension gives you the same debugging experience as the conventional VS Code Debugger. Follow the steps below to start a debug session.
+The VS Code Ballerina extension gives you the same debugging experience as the conventional VS Code Debugger.
```
[the remark this answers is not in the document]
Comment (praveennadarajah) [reply learn/guides/visual-studio-code-extension/debugging.md:36]: I kept the title as it is and added a subsection `Starting a Program Debug Session`.

File: learn/guides/visual-studio-code-extension/debugging.md (hunk 1 above)
Comment (praveennadarajah) [reply learn/guides/visual-studio-code-extension/debugging.md:45]: Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/3348/commits/39614b328067194a4139d08b8e5ca08434260425

File: learn/guides/visual-studio-code-extension/debugging.md (hunk 2 above)
Comment (praveennadarajah) [reply learn/guides/visual-studio-code-extension/debugging.md]: Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/3348/commits/39614b328067194a4139d08b8e5ca08434260425

**Comment by praveennadarajah** [comment]
> @praveennadarajah Shall we add a section on debugging individual tests using the newly added codelens support?

Fixed with https://github.com/ballerina-platform/ballerina-dev-website/pull/3348/commits/0b231ef77da2016885c09155cd70dea5351aaaae

**Comment by praveennadarajah** [comment]
Changes are added with https://github.com/ballerina-platform/ballerina-dev-website/pull/3385

**Pull request closed without merging**